modules/ROOT/pages/design-add-api-dependency.adoc

```diff
@@ -52,6 +52,25 @@ Even if your projects have not been identified as requiring action, you can proa
 [fix-gov-issues]
 == Fix Governance Conformance Issues
 
+When your API fails conformance to rulesets your organization validates against, you get an Email notification:
+
+. Click *Fix me* on your email to be redirected to the Exchange API asset Conformance Status page.
+.. If the latest version of your API has an editable branch, a pop-up appears where you can see the missing rulesets. Click *Add all rulesets*. +
+_Result_: The rulesets are added to your project.
+.. If the latest version of your API doesn't have an editable branch, a pop-up appears where you can select an editable branch on your project, or create a new branch from your current version to add the missing rulesets. 
```

Comment:
The same here, _If the latest version..._ should be replaced with _If the asset version doesn't have an editable branch in the API Designer project..._